test/boost/database_test.cc

```diff
@@ -972,9 +972,9 @@ SEASTAR_THREAD_TEST_CASE(read_max_size) {
                         }
                     } catch (std::runtime_error& e) {
                         if (should_throw) {
-                            testlog.trace("Exception thrown, as expected: {}", e);
+                            testlog.trace("Exception thrown, as expected: {}", e.what());
```

Comment:
This is not identical to printing `e` itself, because printing the `e` object will also print its type. We can live with this for the duration of the switch.

Reply:
thank you for pointing this out. i was under the wrong impression that we were formatting using {fmt}. yeah, if we use the formatter in seastar, it uses RTTI to print the type as well. but after switching to {fmt}, we might need to specialize for `runtime_error` and some other types we are interested in, instead of specializing `std::exception`. as {fmt} already do this.